baseband/vlbi_base/base.py

```diff
@@ -314,7 +319,7 @@ def get_time1(self):
     def size(self):
         """Number of samples in the file."""
         return int(round((self.stop_time - self.start_time).to(u.s).value *
```

Comment:
Use `sample_rate` so you can just do `int(round((self.stop_time - self.start_time) * self.sample_rate))`

Reply:
I'm not sure if this is due to my environment (I'm on Py3.5.2, running numpy 1.14.0 and astropy 2.0.3), but this requires `np.round` and decomposing the units before casting to `int`, i.e.
```
        return int(np.round(((self.stop_time - self.start_time) *
                             self.sample_rate).decompose()))
```